tests/management/group/test_view.py

```diff
@@ -311,6 +326,46 @@ def test_add_group_roles_system_policy_create_success(self):
         self.assertEqual(roles[0].get('description'), self.role.description)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_system_flag_update_on_add(self):
+        """Test that adding a role to a platform_default group flips the system flag."""
+        url = reverse('group-roles', kwargs={'uuid': self.defGroup.uuid})
+        client = APIClient()
+        test_data = {'roles': [self.roleB.uuid, self.dummy_role_id]}
+
+        response = client.post(url, test_data, format='json', **self.headers)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertFalse(Group.objects.filter(uuid=self.defGroup.uuid).first().system)
+
+    def test_system_flag_update_on_remove(self):
+        """Test that removing a role from a platform_default group flips the system flag."""
+        url = reverse('group-roles', kwargs={'uuid': self.defGroup.uuid})
+        client = APIClient()
+        url = '{}?roles={}'.format(url, self.roleB.uuid)
+
+        self.policy.roles.add(self.roleB)
+        self.policy.save()
+
+        response = client.delete(url, format='json', **self.headers)
+
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertFalse(Group.objects.filter(uuid=self.defGroup.uuid).first().system)
+
+
+    def test_system_flag_update_on_rename(self):
```

Comment:
Is there a difference in what this is supposed to test vs `test_update_platform_default_group_success`?

Reply:
Nope, I don't think so. Just got lost in the code and wrote the same test twice, I think. Pulled it out.